tools/ansible/roles/dockerfile/templates/supervisor.conf.j2

```diff
@@ -12,9 +12,10 @@ directory = /awx_devel
 {% else %}
 command = nginx -g "daemon off;"
 {% endif %}
-autostart = true
```

Comment:
removed because it's the default... just want to get that stated as a sanity check.